packages/perseus/src/widgets/interactive-graphs/backgrounds/axis-ticks.tsx

```diff
@@ -83,13 +91,55 @@ const XGridTick = ({x, graphInfo}: {x: number; graphInfo: GraphDimensions}) => {
     const x2 = xPosition;
     const y2 = yPosition - tickSize / 2;
 
+    // Adjust the Y position of the x-axis labels based on
+    // whether the x-axis is above, within, or below the graph
+    const yAdjustment =
+        range[Y][MAX] < 0 ? -tickLabelSize : tickLabelSize * 1.75;
+
+    // Adjust the X position of the x-axis labels based on
+    // whether the label is positive or negative, in order to
+    // account for the width of the negative sign
+    const xAdjustment = x < 0 ? -2 : 0;
+
+    // Apply the adjustments to the x and y positions for the text
+    const xPositionText = xPosition + xAdjustment;
+    const yPositionText = yPosition + yAdjustment;
+
     return (
-        <g className="x-axis-ticks">
-            <line x1={x1} y1={y1} x2={x2} y2={y2} style={tickStyle} />
+        <g className="tick">
+            <line x1={x1} y1={y1} x2={x2} y2={y2} className="axis-tick" />
+            {
+                <text
+                    className="axis-tick-label"
+                    style={{fontSize: tickLabelSize}}
+                    textAnchor="middle"
+                    x={xPositionText}
+                    y={yPositionText}
+                >
+                    {x.toString()}
+                </text>
+            }
         </g>
     );
 };
 
+// Determines whether to show the label for the given tick
+// Currently, the only condition is to hide the label at -1
+// on the y-axis when the x-axis is within the graph
+export const shouldShowLabel = (
+    number: number,
+    range: [Interval, Interval],
+) => {
+    let showLabel = true;
+
+    // If the x-axis is within the graph and the y-axis is at -1, hide the label
+    if (range[X][MIN] < -1 && range[X][MAX] > 0 && number === -1) {
+        showLabel = false;
+    }
+
```

Comment:
I split this into a separate function as I had a feeling that we may come up with more scenarios in the future for when we may want to conditionally hide labels. It also helps keep the code a little more organized.